{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```diff
@@ -51,7 +64,7 @@ RUN --mount=type=cache,uid=$UID,gid=$GID,target=/home/user/.cache/pypoetry/ \
 
 
 
-FROM base as ci
```

Comment:
I'd prefer if we can bring the `ci` image as close to the `app` image as possible. Right now there are some pretty big changes between the two, and so something that works on the CI may not work at all when you actually build and deploy it. Can this also be based off of `base` like `app` is?

(Before answering this, it's probably better to see my other comment about the base image of the `app` stage.)

Reply:
This is not possible, I think - you'll need Poetry and `build-essentials` in the `ci` image. I do understand the concern but I think there's no real risk there if you agree with [my response to the comment regarding the exclusion of `build-essential` from the `app` image](https://github.com/radix-ai/poetry-cookiecutter/pull/205#discussion_r1401983231).

Reply:
I'm more or less convinced by your answer, but I still have an inkling there's something else I'm missing. Let me get back to you on this!

Reply:
Maybe the fault in my reasoning will come to you in a dream, or something of the sort! Do let me know if anything pops up.

Reply:
I'm satisfied!